ci/check-protobuf-against-stable.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+eval "$(dev-env/bin/dade assist)"
+
+readonly BUF_IMAGE_TMPDIR="$(mktemp -d)"
+trap 'rm -rf ${BUF_IMAGE_TMPDIR}' EXIT
+
+readonly CURRENT_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
```

Comment:
This will generally not do what you expect. On PRs, the current branch is a garbage one created by Azure, usually named just `merge` or something like that. If you want to log the source branch (I can't really see a reason to), you can use `SYSTEM_PULLREQUEST_SOURCEBRANCH`.

Reply:
b9005ea